src/views/InteractiveMap/Card.vue

```diff
@@ -309,6 +314,19 @@ export default Vue.extend({
       }
     }
   },
+  mounted() {
```

Comment:
These will need to be reordered when merged into devel, because of the new property order rule, so just remember to run linter before pushing